pkg/upgrade/rollout.go

```diff
@@ -0,0 +1,60 @@
+package upgrade
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/threefoldtech/zos/pkg/environment"
+)
+
+const defaultRolloutConfigURL = "https://raw.githubusercontent.com/threefoldtech/zos/development_rollout_update/pkg/upgrade/config.json"
+
+// RolloutConfig is the configuration for A/B testing before zos update
+type RolloutConfig struct {
+	TestFarms     []uint32 `json:"test_farms"`
+	SafeToUpgrade bool     `json:"safe_to_upgrade"`
```

Comment:
I know this is reflecting the current specs, but should consider specifying the version instead of having just a bool allowing the "latest" whatever that latest is?